SocialFilter/sections/specific.txt

```diff
@@ -57,6 +57,8 @@ math-fun.net,polaris-official.com,apexheadline.com,aucfan.com,sist8.com,spaceshi
 !
+www.yna.co.kr##button.kkt
+yna.co.kr##[class*="-share"]
```

Comment:
I got why you use it without tag, but such things are better explained in the description next time.